src/components/App/App.css

```diff
@@ -0,0 +1,8 @@
+body {
+    padding-left:30px;
+}
+
+.center {
```

Comment:
What is this applied to? I don't see any other changes in here adding a center class.